LabManual.md

```diff
@@ -205,3 +205,132 @@ Click **Search** to search for all documents.
+
+Navigate to https://www.luis.ai and sign in using your Microsoft account.  You should be redirected to a list of your LUIS applications at https://www.luis.ai/applications.  We will create a new LUIS app to support our bot.  
+
+> Fun Aside: Notice that there is also an "Import App" next to the "New App" button on [the current page](https://www.luis.ai/applications).  After creating your LUIS application, you have the ability to export the entire app as JSON, and check it into source control.  This is a recommended best practice so you can version your LUIS models as you version your code.  An exported LUIS app may be re-imported using that "Import App" button.  If you fall behind during the lab and want to cheat, you can click the "Import App" button and import the [LUIS model from the Finished section](./Finished/LUIS/CognitiveServicesTutorialLuisModel.json).  
+
+From https://www.luis.ai/applications, click the "New App" button.  Give it a name (I chose "PictureBotLuisModel") and set the Culture to "English".  You can optionally provide a description.  Click the dropdown to select an endpoint key to use, and if the LUIS key that you created on the Azure portal at the beginning of this lab is there, select it.  Then click "Create".  
+
+![LUIS New App](./assets/LuisNewApp.jpg) 
+
+You will be taken to a Dashboard for your new app.  The App Id is displayed; note that down for later as your LUIS App ID.  Then click "Create an intent".  
+
+![LUIS Dashboard](./assets/LuisDashboard.jpg) 
+
+We want our bot to be able to do the following things:
++ Search/find pictures
++ Share pictures on social media
++ Order prints of pictures
++ Greet the user (although this can also be done other ways as we will see later)
+
+Let's create intents for the user requesting each of these.  Click the "Add intent" button.  
+
+Name the first intent "Greeting" and click "Save".  Then give several examples of things the user might say when greeting the bot, pressing "Enter" after each one.  After you have entered some utterances, click "Save".  
+
+![LUIS Greeting Intent](./assets/LuisGreetingIntent.jpg) 
+
+Now let's see how to create an entity.  When the user requests to search the pictures, they may specify what they are looking for.  Let's capture that in an entity.  
+
+Click on "Entities" in the left-hand column and then click "Add custom entity".  Give it an entity name "facet" and entity type "Simple".  Then click "Save".  
+
+![Add Facet Entity](./assets/AddFacetEntity.jpg) 
+
+Now click "Intents" in the left-hand sidebar and then click the yellow "Add Intent" button.  Give it an intent name of "SearchPics" and then click "Save".  
+
+Now let's add some sample utterances.  People might search for pictures in many ways.  Feel free to use some of the utterances below, and add your own wording for how you would ask a bot to search for pictures.  
```

Comment:
Include parenthetical small def'n of utterances. i.e. "sample utterances (simple sentences you think users might say that the bot should understand)"